scripts/create-package/plopfile.ts

```diff
@@ -49,12 +50,26 @@ module.exports = (plop: NodePlopAPI) => {
         default: true,
         when: answers => answers.target === 'node', // react always has tests
       },
+      {
+        type: 'confirm',
+        name: 'hasStorybook',
+        message: 'Create storybook package scaffolding?',
+        default: true,
+        when: answers => answers.target === 'react',
+      },
     ],
 
     actions: (answers: Answers): Actions => {
-      const { packageName, target, hasTests } = answers;
+      // hasTests should default to true / however under
+      // react package it get's set to undefined.
+      // we default hasTests to true in that scenario
+      answers = { ...answers, ...{ hasTests: answers.hasTests !== undefined ? answers.hasTests : true } };
```

Comment:
nit, a bit simpler way:
```suggestion
      const { hasTests = true } = answers;
      answers = { ...answers, hasTests };
```